src/main/AndroidManifest.xml

```diff
@@ -376,6 +376,13 @@
               android:exported="true">
     </activity>
 
+    <activity android:name=".WebxdcStoreActivity"
+              android:label=""
+              android:theme="@style/TextSecure.LightTheme"
+              android:configChanges="touchscreen|keyboard|keyboardHidden|orientation|screenLayout|screenSize|uiMode"
+              android:exported="true">
```

Comment:
`exported` means that other apps can launch this activity; this isn't necessary here